gtfs_realtime_translators/translators/path_rail.py

```diff
@@ -0,0 +1,135 @@
+import json
+import math
+
+import pendulum
+
+from gtfs_realtime_translators.factories import FeedMessage, TripUpdate
+
+
+class PathGtfsRealtimeTranslator:
+    TIMEZONE = 'America/New_York'
+
+    ROUTE_ID_LOOKUP = {
+        '#4D92FB': '859',
+        '#65C100': '860',
+        '#FF9900': '861',
+        '#D93A30': '862',
+        '#4D92FB,#FF9900': '1024'}
```

Comment:
I'm confused about what is happening here. Is it valid to set two colors here?

Reply:
For late night and weekend service along the 33rd Street - Hoboken line, the service combines two routes. PATH is choosing to denote both of those routes in this manner in their realtime feed.

https://www.panynj.gov/path/en/schedules-maps.html

From the [test json](https://github.com/Intersection/gtfs-realtime-translators/blob/path-arrivals/test/fixtures/path_rail.json)
```
{
      "consideredStation": "GRV",
      "label": "ToNJ",
      "target": "JSQ",
      "messages": [
        {
          "secondsToArrival": 167,
          "arrivalTimeMessage": "3 min",
          "lineColor": "#4D92FB,#FF9900",
          "headSign": "Journal Square via Hoboken",
          "lastUpdated": "2020-02-22T15:45:59.562598-05:00"
        }
      ]
    },
```